tests/core/models.py

```diff
@@ -1,20 +1,23 @@
 from itertools import count
 import uuid
 
-from django.contrib.auth.models import User
 from django.db import models
 
 from tastypie.utils import now, aware_datetime
+from tastypie.compat import AUTH_USER_MODEL
 
 
 class DateRecord(models.Model):
     date = models.DateField()
     username = models.CharField(max_length=20)
     message = models.CharField(max_length=20)
 
+    class Meta:
+        app_label = 'core'
+
 
 class Note(models.Model):
-    author = models.ForeignKey(User, related_name='notes', blank=True, null=True)
+    author = models.ForeignKey(AUTH_USER_MODEL, related_name='notes', blank=True, null=True)
```

Comment:
Since these models get used during the customuser tests, 1.11 was throwing an error because it noticed the manager was pointing to the wrong model.